pxd.h

```diff
@@ -19,7 +19,7 @@
 #define PXD_DEV  	"pxd/pxd"		/**< block device prefix */
 #define PXD_DEV_PATH	"/dev/" PXD_DEV		/**< block device path prefix */
 
-#define PXD_VERSION 10				/**< driver version */
+#define PXD_VERSION 11				/**< driver version */
```

Comment:
If pxd version is marked 11, then 2.7 porx px-storage will try using memory mapped interface to talk to kernel.

Reply:
There's a porx side version change accompanying this to prevent it in pxdriver.h

Reply:
here: https://github.com/portworx/porx/pull/7410